docs/BasicCommunication/OnEventChanged/index.md

```diff
@@ -18,20 +18,18 @@ SDL uses `OnEventChanged` notification to appropriately manage the hmiLevel and
 !!! MUST
 1. Send notification with appropriate parameter value when active call on HMI has been started/ended.
 
-2. Send [SDL.OnAppDeactivated](../../sdl/onappdeactivated) to the active app when the phone call is started.
-
-3. Resume the applications to their original state prior to the phone call event in the HMI when the event ends (see note below).
+2. Resume the applications to their original audio streaming state prior to the phone call event in the HMI when the event ends (see note below).
 !!!
 
 !!! NOTE
-SDL does not send BC.ActivateApp or BC.OnResumeAudioSource to HMI after the phone call is ended.
+If the HMI wants to leave the app screen when the PHONE_CALL event occurs, the app's HMI state can be changed accordingly using the `BC.OnAppDeactivated(appID)` and `BC.OnAppActivated(appID)` notifications.
 !!!
 
 Upon receiving `OnEventChanged(PHONE_CALL)`, SDL will:
 
 |isActive|Result|
 |:-------|:-----|
-|true|Change the HMI state of all media applications currently either in FULL or LIMITED to (BACKGROUND, NOT_AUDIBLE) and all navi/projection apps to (LIMITED, NOT_AUDIBLE, STREAMABLE)|
+|true|Move all apps with audioStreamingState AUDIBLE to NOT_AUDIBLE|
 |false|Return applications to the same HMI state they had prior to the event|
```

Comment:
```suggestion
|false|Return applications to the same HMI state they had prior to the event (if not changed during the event)|
```

Reply:
@jacobkeeler -san,
Basically, we will accept your suggestion.
However, we have two question.
1.
Our proposal is to apply to only case of "isActive = true".
Therefore, we assume that the case of "is Active = false" is not applicable.
2.
We left [this comment](https://github.com/smartdevicelink/sdl_hmi/pull/576#issuecomment-880328403).
If need to modify this, we think that it is necessary to add the case of changing the last application during the event.

For these reasons, we ccouldnot make a commit it.
If you have some idea, could you please teach us.
Thank you.

Reply:
1. Hmmm, thinking on it, this is actually more of an existing issue with the documentation, not necessarily tied to this proposal. Perhaps I'll create a separate PR for this, since it would probably affect the other states as well.